bin/apisix

```diff
@@ -659,14 +689,21 @@ local function init()
         with_module_status = false
     end
 
+    local with_module_reqstas = false
+    if or_ver and or_ver:find("ngx_http_reqstat_module", 1, true) then
```

Comment:
you should add link for this module, and Tengine compiles this module by default

Reply:
according your compile script .travis/linux_tengine_runner.sh, this module is enabled by default.